sw/host/opentitanlib/src/transport/chip_whisperer/board.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+pub trait Board {
+    const VENDOR_ID: u16 = 0x2b3e;
+    const PRODUCT_ID: u16;
+
+    // Pins needed for SPI on the Chip Whisperer board.
+    const PIN_CLK: &'static str;
+    const PIN_SDI: &'static str;
+    const PIN_SDO: &'static str;
+    const PIN_CS: &'static str;
+    // Pins needed for reset & bootstrap on the Chip Whisperer board.
+    const PIN_TRST: &'static str;
+    const PIN_POR_N: &'static str;
+    const PIN_SW_STRAP0: &'static str;
+    const PIN_SW_STRAP1: &'static str;
+    const PIN_SW_STRAP2: &'static str;
+    const PIN_TAP_STRAP0: &'static str;
+    const PIN_TAP_STRAP1: &'static str;
+}
+
+pub struct Cw310 {}
+
+impl Board for Cw310 {
+    const PRODUCT_ID: u16 = 0xc310;
+
+    // Pins needed for SPI on the Chip Whisperer board.
+    const PIN_CLK: &'static str = "USB_SPI_SCK";
+    const PIN_SDI: &'static str = "USB_SPI_COPI";
+    const PIN_SDO: &'static str = "USB_SPI_CIPO";
+    const PIN_CS: &'static str = "USB_SPI_CS";
+    // Pins needed for reset & bootstrap on the Chip Whisperer board.
+    const PIN_TRST: &'static str = "USB_A13";
+    const PIN_POR_N: &'static str = "USB_A14";
+    const PIN_SW_STRAP0: &'static str = "USB_A15";
```

Comment:
Probably this should be removed and the code should use the json definition. I think (but I am not sure) that
```rust
const PIN_SW_STRAP0: &'static str = "PIN_SW_STRAP0";
```
would work and could be a general ChipWhisperer setting, not just CW310 specific. But I haven't looked at the code enough to be sure.

Reply:
I think these names have to be resolvable within the backend (ie: by either SCHEMATIC_PIN_NAMES or SAM3X_PIN_NAMES).  The definitions from the JSON files get resolved at the TransporWrapper layer.

These pin definitions are needed to configure low-level while initializing the backend and I don't think the TransportWrapper pin resolution is available at that time.  I'd be happy to be proven wrong about it though.

Reply:
Yes, you are right @cfrantz, that's why I didn't want to touch that in this PR. But I'm working on another PR to refactor that and centralize the pin definition in the json file.